Electron/src/__tests__/components/Sidebar/AddSongPlayListAccordion.test.tsx

```diff
@@ -188,10 +188,15 @@ test('AddSongPlaylistAccordion submit song correct', async () => {
   const dropdown = component.getByTestId('select-genre');
 
   // Find the file input element by it's name or other suitable selector
-  const fileInputElement = component.getByTestId('sidebar-file-input');
+  const fileInputElement = component.getByTestId(
+    'sidebar-file-input',
+  ) as HTMLInputElement;
 
-  // Create a sample file
-  const file = new File(['(⌐□_□)'], 'sample.mp3', { type: 'audio/mp3' });
+  // Create a sample files
+  const mp3File = new File(['(⌐□_□)'], 'sample.mp3', { type: 'audio/mp3' });
+  const wavFile = new File(['(⌐□_□)'], 'sample.wav', { type: 'audio/wav' });
+  const flacFile = new File(['(⌐□_□)'], 'sample.wav', { type: 'audio/wav' });
```

Comment:
Maybe this is a typo, shouldn't it be audio/flac?

Reply:
Sorry my bad. Yes it is a typo. I will fix it.